### PR TITLE
Week 3 contents: ...

### PR DESCRIPTION
**Pull request by chanjunweimy**
... Added things-to-do, tutorial2, and lecture3 of week 3 contents into the week3.html

### CODE REVIEW

File: contents/week3.html [hunk 1]
```diff
@@ -93,22 +193,124 @@ <h3 id="ilo-content-week3" class="buttoned"><span>Intended Learning Outcomes</sp
...
+								</li>
+								<li>
+									<a target="_blank" href="http://scesm04.upb.de/case-study-2/requirements.pdf">
+									A weather-related scientific software</a> (source: NASA) - 
+									Because of the nature of the software, the specification is very technical, 
+									detailed, and completely text-based.
+								</li>
+							</ul>
+						</li>
+                        <li>
```
Comment (damithc) [inline contents/week3.html]: Too much indentation. Each indentation level should be 4 spaces.

**Comment by damithc** [comment]
Minor change requested.

**Comment by damithc** [comment]
Also, pre-tutorial activities section is messed up. Missing section headings? If there are no section headings in the original schedule, you might need to add them.

**Comment by chanjunweimy** [comment]
I have changed tabs to 4 spaces and made the Pre-tutorial activities slightly bigger. :)

File: contents/week3.html [hunk 2]
```diff
@@ -1,69 +1,164 @@
...
+		<div class="things-to-do things-to-do-week3">
+            <h3>Start product design:</h3>
+            <div>
+                After you have at least a tentative team, it is time to start thinking about product design.
+
+				Tutorial 1 exercise gave you a 'Yes' pile of user stories. Use that as a baseline 
+				for your product design. Note that what you have is just a collection of user stories 
+				which may not necessarily translate into a good product. You might have to do more 
+				reshuffling of user stories before they form a cohesive product that appeals to the 
+				target users. More about product design will be covered in a future lecture.
```
Comment (damithc) [inline contents/week3.html]: I still see too much indentation here, and other places.

**Comment by chanjunweimy** [comment]
Is it ok now? :)

**Comment by damithc** [comment]
Remove this link. It doesn't work. 
image
This part is messed up:
image
This should be a regular link. Don't format it as a link to embedded content.
image
These two panels don't open when I click on the '+' button:
image

**Comment by chanjunweimy** [comment]
Is it ok now? :)

**Comment by damithc** [comment]
This content arrangement does not match week 2
image

**Comment by damithc** [comment]
Pre-tutorial learning activities are still messed up. Compare with week 2.
For some reason, week 3 '+' button seem to behave differently from other weeks. 
On a related note, try to reach the final version in fewer tries.

**Comment by chanjunweimy** [comment]
I dun understand what do you mean by '+' button "behave differently" but I think I solve the pre-tutorial learning activities already and arrange the content to be same with week 2.

**Comment by damithc** [comment]
This is not a good title. It has two sentences. Come up with a shorter one similar to the first one.
image
The problem with the '+' button is ok now.

**Comment by chanjunweimy** [comment]
I have renamed the title to refactoring.

**Pull request merged**